File: SaoriChef/Class/Main/Controller/MainViewController.m
//
//  MainViewController.m
//  SaoriChef
//
//  Created by さくら　さおり on 2019/12/28.
//  Copyright © 2019 やまぐち　さおり. All rights reserved.
//

#import "MainViewController.h"
#import "MainView.h"

@interface MainViewController ()

@property (nonatomic) MainView *mainView;
@end

@implementation MainViewController

- (void)loadView
{
    self.mainView = [MainView mainView];
    [self setView:self.mainView];
}

- (void)viewDidLoad {
    [super viewDidLoad];
}

@end

File: SaoriChef/Class/Main/Controller/MainViewController.swift
//
//  MainViewController.swift
//  Kakeibo
//
//  Created by 長沼 真宏 on 2019/12/27.
//  Copyright © 2019 長沼 真宏. All rights reserved.
//

import UIKit

class MainViewController: UIViewController {

    var mainVC:MainView!
    
    override func loadView() {
        self.mainVC = MainView.instance()
        self.view = self.mainVC
    }
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
}
